gamestonk_terminal/forex/fx_view.py

```diff
@@ -228,7 +229,7 @@ def create_order(accountID, instrument, other_args: List[str]):
         "--price",
         dest="price",
         action="store",
-        type=float,
+        type=str,
```

Comment:
you probably should check that the string parsed can be converted to a valid price.

I would personally keep the float and even use the method check_non_negative, and then when you use it instead of using it with `ns_parser.price` you can do `str(ns_parser.price)` if you want the string.

That way you ensure the user is providing a valid number

Reply:
I changed it to a string when I encountered a price precision error with some floats due to them being stored with trailing nines. It changed 1.20849 to something like 1.208499999999. I checked a few more times after changing it and it didn't seem to happen with strings. Once it's sent to Oanda they check that it's a valid price and return a V20Error if it's invalid, so I think having an extra check before sending it is unnecessary since the feedback to the user would happen at roughly the same time.

Reply:
You could potentially use 
```
str(round(ns_parser.price, 2))
```

to round with 2 decimal places.

The reason why it works with strings is because the user will not add a lot of decimal places. However, in this situation, the main problem is that the user can use `--price HELLO` and this will break because we didn't check that the value provided was a number, if it makes sense.

Reply:
It just printed the invalid price error without crashing with regular strings like hello, I went ahead and changed it to use floats and round though.